### ISSUE TITLE
Working with Laravel 5 Auth.

### ISSUE THREAD

**Issue by iamraffe** [HipsterJazzbo/LaraParse#9]
Hi,

I am sorry if this is not the place to post this. I am trying to get Parse SDK and Laravel 5 to work together.

I have managed to get the basic setup running, however I don't seem to be able to get the Authentication part of the app to work. I am setting the driver to parse, however this is the first time I am not working with a "normal" database and I don't know exactly how to tell the app that I want to register my users in my Parse DB instead of my standard Homestead DB.

I am pretty sure this is a total noob question but I am stuck so any help would be greatly appreciated as I am quite overwhelmed in my first steps with Parse.

R.

**Comment by nicklee1990** [comment]
It think it'd be helpful if you tried following the instructions then pasted any exceptions/errors you're getting? 

Set up should be pretty straightforward. A few pointers:
- Set your driver to parse.
- In your default Laravel 5 installation you should have an Auth Controller. Just update the 'use....' that to reference Registrar class in the LaraParse package. You may need to amend your login/register forms to supply the correct information
- If you want to override the LaraParse Registrar, just change the default Registrar class to extend the LaraParse registrar and override the methods as required.

**Comment by iamraffe** [comment]
Thanks for helping me out with this.

I was missing that 'use' in AuthController. I had updated it in AuthenticatesAndRegistersUsers.php, but not in that one. I know it was some detail like that.

I am getting this error now, however:
"
ParseException in ParseClient.php line 293:
unauthorized

"

My guess I must have changed some key. Will verify.

Thanks again.

**Comment by iamraffe** [comment]
Indeed I was incorrectly setting my variables in the .env file. 

Now I've got it working.

Thanks again man, you are a lifesaver.

If ever in Madrid, I'll buy you a beer.

**Comment by nicklee1990** [comment]
I'll hold you to that!

**Comment by travisjbeck** [comment]
Hey can you ELI5 how you got this working? I'm having the same problems.  new to laravel.

**Comment by nicklee1990** [comment]
What error are you getting? With a fresh install of Laravel, the Registrar class is no longer present which is probably causing an issue. Effectively you just need to pull the methods out of the LaraParse Registrar and override the relevant methods from the AuthenticatesAndRegistersUsers trait

**Comment by hipsterjazzbo** [comment]
@nicklee1990 I forgot that's no longer included. We should come up with a better out of the box solution to that.
